cnames_active.js

```diff
@@ -1658,6 +1658,7 @@ var cnames_active = {
   "naja": "jiripudil.github.io/Naja",
   "name-my-colors": "domcoleman.github.io/name-my-colors",
   "namelessman": "namelessman.github.io",
+  "namaste" : "namaste-duniya.github.io/js/",
```

Comment:
```suggestion
  "namaste": "namaste-duniya.github.io/js",
```
Remove extra space, remove trailing slash (you have maintainer edits disabled, so I cannot do this myself)